### PR TITLE
feat: Add 2021Q3 roadmap

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

### CODE REVIEW

File: blog/2021-08-06-2021q3-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+In the past year:
+
+- go-storage API has been more and more stable, with no big break changes anymore.
+- Up to 21 services have been or are under development.
+- BeyondTP has released several beta versions.
+
+> We have many ideas so far, all of them are listed in [BeyondStorage 2021Q3 Roadmap Draft](https://forum.beyondstorage.io/t/beyondstorage-2021q3-roadmap-draft/164), in this post, we only discuss our roadmap.
+
+Now we have a solid foundation to support us to go higher and further, we have to think about where should BeyondStorage going on.
```
Comment (Prnyself) [inline blog/2021-08-06-2021q3-roadmap.md:24]:
```suggestion
Now we have a solid foundation to support us to go higher and further, we have to think about where BeyondStorage should go.
```

File: blog/2021-08-06-2021q3-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Roadmap
+
+### More Services Connected
+
+For now, we have 21 supported services: 9 of them are stable, 1 of them is beta, 10 of them are under development.
```
Comment (Prnyself) [inline blog/2021-08-06-2021q3-roadmap.md:30]: 9+1+10 = 20？

File: blog/2021-08-06-2021q3-roadmap.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+In 2021Q3, we will expand them to 30 services:
+
+- Already implemented services will become stable.
+- And we will implement 9 more services.
```
Comment (Prnyself) [inline blog/2021-08-06-2021q3-roadmap.md:35]:
```suggestion
- Services already implemented will become stable.
- 9 more services will be implemented.
```

File: blog/2021-08-06-2021q3-roadmap.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+### More Releases for BeyondTP
+
+BeyondTP is a neutral data migration service that is built upon go-storage. In the past several months, our development speed is too slow. In 2021Q3, we will accelerate iteration speed.
+
+After 2021Q3, we will release a usable version of BeyondTP.
+
+### More Scene Expansion
+
+In 2021Q3, we will expand more scenes of BeyondStorage.
```
Comment (Prnyself) [inline blog/2021-08-06-2021q3-roadmap.md:47]:
```suggestion
In 2021Q3, we will expand more scenes for BeyondStorage.
```

File: blog/2021-08-06-2021q3-roadmap.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+First of all, we will try to build BeyondFS: a high-performance, POSIX-ish File System based on go-storage. In 2021Q3, we plan to build a first beta release version of BeyondFS.
+
+Then, we will build an FTP server based on go-storage: BeyondFTP. This service could behave like an FTP gateway of all storage services.
```
Comment (Prnyself) [inline blog/2021-08-06-2021q3-roadmap.md:51]:
```suggestion
Then, we will build an FTP server based on go-storage: BeyondFTP. This service could behave like an FTP gateway for all supported storage services.
```

File: blog/2021-08-06-2021q3-roadmap.md (hunk 2 above)
Comment (Xuanwo) [reply blog/2021-08-06-2021q3-roadmap.md:30]:
```suggestion
For now, we have 21 supported services: 9 of them are stable, 1 of them is beta, 11 of them are under development.
```

**Review by Prnyself — APPROVED** [review]
LGTM

**Comment by Xuanwo** [comment]
ping @abyss-w @JinnyYi @xiongjiwei for another review.

**Review by JinnyYi — APPROVED** [review]

**Review by abyss-w — APPROVED** [review]

**Review by xiongjiwei — APPROVED** [review]

**Pull request merged**